.gitignore

```diff
@@ -1,2 +1,3 @@
 venv/
```

Comment:
```suggestion
*venv/
```
The Python venv docs actually recommend `.venv` so it is best to be ready for both.